package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "web3-onboard-monorepo",
-  "version": "2.0.2",
+  "version": "2.1.1",
```

Comment:
```suggestion
  "version": "2.1.0",
```

Reply:
It's "2.1.1" in the other PR. Do we increment this also?

Reply:
@taylorjdawson the previous release(last week) was `2.1.0` - https://github.com/blocknative/web3-onboard/pull/1087

Reply:
okay can we do `2.2.0` then?

Reply:
@Adamj1232 wondering if we should always use minor instead of the patch version for the base version? I think patch is reserved for bug fixes? and minor adds new functionality

Reply:
@taylorjdawson ahh just saw your comment on the other PR lol
Totally good with me and makes sense